Makefile

```diff
@@ -101,9 +101,9 @@ release: POST_SDIST=upload
 release: bdist sdist ## Make a wheel + source release on PyPI
 
 
-docker_images: docker_image_enterprise-gateway docker_image_nb2kg docker_image_hadoop-spark ## Build docker images
```

Comment:
Any reason we have different styles on the docker image names some with underscore others with mix of underscore and dash ?

Reply:
(@lresende - Our previous conversation was removed due to recent update).  
I found a style guide that stated that makefile target names should have words separated by hyphens - so I pushed a corresponding update.  I agree that it was a bit odd to have a mixture.   My previous convention was to preserve the format of the artifact name (e.g., `enterprise-gateway`) but use underscores to separate target words that weren't part of the artifact (e.g., `docker_image_`) and thus - `docker_image_enterprise-gateway`.  But this is now `docker-image-enterprise-gateway`.